django_cloud_deploy/cli/prompt.py

```diff
@@ -1136,6 +1138,81 @@ def _validate(self, s: str):
                               '"test@example.com"').format(s))
 
 
+class DjangoSettingsPathPrompt(StringTemplatePrompt):
+    """Allow the user to enter the settings file path of a Django project."""
+
+    PARAMETER = 'django_settings_path'
+    MESSAGE = ('{} Enter the settings file path you want to use for deployment')
```

Comment:
This isn't completely clear to me. Maybe explain it and we can work on the wording a bi t?

Reply:
I added more comments for this class.

Reply:
I mean that the message to the user isn't completely clear. I can't think of anything much better:

```suggestion
    MESSAGE = ('{} Enter the path of the Django settings file that should be used for deployment')
```

Reply:
Done.